### PR TITLE
Fix version information

### PR DESCRIPTION
**Pull request by lcobucci**
Since `Doctrine\ORM\Version` is now deprecated it shall not be updated on future releases.

This ensures that our CLI tool will present the correct version number.

### CODE REVIEW

File: lib/Doctrine/ORM/Tools/Console/ConsoleRunner.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 use Doctrine\DBAL\Tools\Console as DBALConsole;
 use Doctrine\ORM\EntityManagerInterface;
 use Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper;
-use Doctrine\ORM\Version;
+use PackageVersions\Versions;
```
Comment (greg0ire) [inline lib/Doctrine/ORM/Tools/Console/ConsoleRunner.php]:
```suggestion
use OutOfBoundsException;
use PackageVersions\Versions;
```

File: lib/Doctrine/ORM/Tools/Console/ConsoleRunner.php [hunk 2]
```diff
@@ -70,10 +70,11 @@ public static function run(HelperSet $helperSet, array $commands = []) : void
      * @param array                                       $commands
      *
      * @return \Symfony\Component\Console\Application
+     * @throws \OutOfBoundsException
```
Comment (greg0ire) [inline lib/Doctrine/ORM/Tools/Console/ConsoleRunner.php]:
```suggestion
     * @throws OutOfBoundsException
```

**Review by greg0ire — COMMENTED** [review]
Looks good, I added suggestions to fix the build

**Review by greg0ire — APPROVED** [review]

**Comment by alcaeus** [comment]
> Since `Doctrine\ORM\Version` is now deprecated it shall not be updated on future releases.

This is a significant BC break: the only purpose of the class is to expose the currently installed version so it be updated on subsequent releases.

Since the version is exposed as a constant, there's no easy way to dynamically generate this. A workaround could be to create the class dynamically in an autoloader file (reading the version with the help of the ocramius/package-versions package), but that could be considered hacky. @lcobucci would you be open to merging that solution?

**Comment by Ocramius** [comment]
> This is a significant BC break: the only purpose of the class is to expose the currently installed version so it be updated on subsequent releases.

It's not a BC break: the version is frozen, and new consumers (relying on `version_compare()`) will immediately see the deprecation. Old consumers will be fine (no changes there).

**Pull request merged**